lightly_studio/src/lightly_studio/resolvers/sample_resolver.py

```diff
@@ -155,10 +155,8 @@ def get_all_by_dataset_id(  # noqa: PLR0913
                 SampleEmbeddingTable,
                 col(SampleTable.sample_id) == col(SampleEmbeddingTable.sample_id),
             ).where(SampleEmbeddingTable.embedding_model_id == embedding_model_id)
-    else:
-        samples_query = samples_query.order_by(
-            col(SampleTable.created_at).asc(), col(SampleTable.sample_id).asc()
-        )
+    elif sample_ids is None:
```

Comment:
Why is the `None` check neeed?

Reply:
The function does not preserve the order in `sample_ids`. Before we were always ordering be created_at (if not by embeddings). Let's not change it in this PR - can you remove the None check?